network-api/networkapi/wagtailpages/pagemodels/customblocks/aside_fields.py

```diff
@@ -0,0 +1,7 @@
+from .. import customblocks
+
+
+aside_fields = [
+    ('linkbutton', customblocks.LinkButtonBlock()),
+    ('spacer', customblocks.BootstrapSpacerBlock()),
+]
```

Comment:
*Question* Don't we need some kind of text block for the aside content? Or is this added later?

Reply:
That's coming later :+1: